hudi-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataFileSystemView.java

```diff
@@ -90,14 +90,14 @@ protected Map<Pair<String, Path>, FileStatus[]> listPartitions(List<Pair<String,
   }
 
   @Override
-  public void reset() {
-    super.reset();
```

Comment:
instead of all this. Can we make this impl just use the writelock here as well? and move on? i.e make the `writeLock` protected. and call the code in `super.reset()` here inline.

Reply:
Fixed.